### PR TITLE
REPL updates

### LINKED ISSUE #112: Autocomplete for schema types in REPL
Implement auto complete for schema types on the CREATE command.

### PR DESCRIPTION
**Pull request by gideonw**
closes #112

### CODE REVIEW

**Review by dburkart — APPROVED** [review]
LGTM, I think this needs rebase after you land the other PR tho?

File: cmd/fossil/client/client.go
Comment (gideonw) [inline cmd/fossil/client/client.go:214]:
```suggestion
```

**Pull request merged**
